synapse/cores/common.py

```diff
@@ -1532,6 +1568,9 @@ def _reqSizeByMeth(self, name):
             raise NoSuchGetBy(name)
         return meth
 
+    def _getTufosByMeth(self, name):
```

Comment:
i wouldn't bother functionalizing this ( just access the dict locally since it's yours, and faster )